scheduler/reconcile_test.go

```diff
@@ -5282,6 +5315,7 @@ func TestReconciler_Disconnected_Client(t *testing.T) {
 		jobVersionIncrement          uint64
 		nodeScoreIncrement           float64
 		disconnectedAllocStatus      string
+		disconnectedAllocStates      []*structs.AllocState
```

Comment:
It ended up a bit repetitive, but I think it's important to highlight that `AllocStates` is also used for scheduling decisions.

Reply:
you could make a little helper function to tidy up the repetition

Reply:
Ah yeah, it was actually simpler to just create a variable with the most common value. Thanks!